src/constraint.jl

```diff
@@ -1,5 +1,5 @@
 """
-Represents a constraint for a [`ContextSensitiveGrammar`](@ref).
+Represents a constraint for a `ContextSensitiveGrammar`.
```

Comment:
I'm not sure why it didn't occur to me earlier, but we could link to `AbstractGrammar` instead. Was this already discussed before?

Reply:
I don't remember this being discussed. I changed it.